python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -28,7 +28,7 @@ class IPrepareApp(IApplication):
     Base class for all applications which can be placed into a prepared\
     state. 
     """
-    _schema = Schema(Version(0, 0), {'hash': SimpleItem(defvalue=None, typelist=['type(None)', 'str'], hidden=1)})
+    _schema = Schema(Version(0, 0), {'hash': SimpleItem(defvalue=None, typelist=['None', 'str'], hidden=1)})
```

Comment:
In that case I'll happily make the jump in this PR, was wanting to avoid changing anything too much as I simply don't want to break things that are otherwise working. I'll give it a shot in another commit